packages/venia-ui/lib/RootComponents/Category/category.gql.js

```diff
@@ -0,0 +1,51 @@
+import gql from 'graphql-tag';
+
+export const GET_CATEGORY_DATA = gql`
+    query category(
+        $id: Int!
+        $pageSize: Int!
+        $currentPage: Int!
+        $onServer: Boolean!
+        $filters: ProductAttributeFilterInput!
+        $sort: ProductAttributeSortInput
+    ) {
+        category(id: $id) {
+            id
+            description
+            name
+            product_count
+            meta_title @include(if: $onServer)
+            meta_keywords @include(if: $onServer)
+            meta_description
+        }
+        products(
+            pageSize: $pageSize
+            currentPage: $currentPage
+            filter: $filters
+            sort: $sort
+        ) {
+            items {
+                # id is always required, even if the fragment includes it
+                id
+                # The following values are used by GalleryItem
```

Comment:
Can we have `GalleryItem` export a fragment and plug that in here?

Reply:
Created a fragment, but unfortunately a bug is preventing us from using it. Added a TODO/comment in eb23ac901